docker/dash/backend/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:14.04
+MAINTAINER Akvo Foundation <devops@akvo.org>
+
+# Configure locales
```

Comment:
we need to make sure that LANG environment is `en_US.UTF8`, we have had issues in the past and we have had to force it e.g. https://github.com/akvo/akvo-flow-services/commit/66cda7452798c8b96d6b8584ef354c8937ee0379